From: =?utf-8?q?Christian_G=C3=B6ttsche?= <cgzones@googlemail.com>
Date: Tue, 5 Sep 2017 11:07:01 +0200
Subject: fix resource leak

The file descriptor used for dup'ing stdout and stderr is not closed.

Found by coverity
---
 src/daemon.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/daemon.c b/src/daemon.c
index 8dc0a9b..22c09a8 100644
--- a/src/daemon.c
+++ b/src/daemon.c
@@ -83,6 +83,8 @@ void daemonize(void)
 		exit(EXIT_FAILURE);
 	}
 
+	close(i);
+
 	umask(027); /* set newly created file permissions */
 
 	/* change running directory */
